docs/source/algorithms/CrossCorrelate-v1.rst

```diff
@@ -9,20 +9,37 @@
 Description
 -----------
 
-Compute the cross correlation function for a range of spectra with
-respect to a reference spectrum.
+Compute the cross correlation function for a range of spectra with respect to a reference spectrum.
 
-.. math:: c_{fg}(r) = \frac{1}{\sigma_f \sigma_g} \int (f(x) - \mu_f) (g(x+r)-\mu_g) dx
+This is use in powder diffraction experiments when trying to estimate the offset (in number of bin indices) of one spectra with respect to another one.
+The input spectra should be in d-spacing and then interpolate on the X-axis of the reference before calculating the cross correlation in bin index.
+The cross correlation function is computed
 
-This is use in powder diffraction experiments when trying to estimate
-the offset of one spectra with respect to another one. The spectra are
-converted in d-spacing and then interpolate on the X-axis of the
-reference. The cross correlation function is computed in the range
-[-N/2,N/2] where N is the number of points.
+.. math:: c_{fg}(r) = \frac{1}{\sigma_f \sigma_g} \int_{XMin}^{XMax} (f(x) - \mu_f) (g(x+r)-\mu_g) dx
+
+where :math:`x` and :math:`r` are bin indices.
+:math:`r` being either half of the number of points in the requested range, ``XMin`` to ``XMax``, of the reference spectrum, or up to `MaxDSpaceShift`, whichever is smaller.
+:math:`f(x)` is the reference spectrum, with :math:`\mu_f` and :math:`\sigma_f` being the mean and variance respectively.
```

Comment:
mean and variance make sense for input Gaussian spectra and also close enough to make sense when it's a single TOF-peak, but what do they mean when the input is an entire powder pattern?

Reply:
It really is the "simple" mean and variance of the y-values. It is a way to make the cross-correlation value more behaved. Those equations can be added or links for the pages in Wikipedia, but I'm not sure that would help people understand. Do you have different wording that can help?

Reply:
not really.